src/Umbraco.Tests.Integration/TestServerTest/UmbracoBuilderExtensions.cs

```diff
@@ -16,39 +17,42 @@ public static class UmbracoBuilderExtensions
         /// </summary>
         /// <param name="builder"></param>
         /// <returns></returns>
-        public static IUmbracoBuilder WithTestCore(this IUmbracoBuilder builder, TestHelper testHelper, LightInjectContainer container,
+        public static IUmbracoBuilder WithTestCore(this IUmbracoBuilder builder, TestHelper testHelper,
+            LightInjectContainer container,
             Action<CoreRuntime, RuntimeEssentialsEventArgs> dbInstallEventHandler)
         {
             return builder.AddWith(nameof(global::Umbraco.Web.Common.Builder.UmbracoBuilderExtensions.WithCore),
-                    () =>
-                    {
-                        builder.Services.AddUmbracoCore(
-                            builder.WebHostEnvironment,
-                            container,
-                            typeof(UmbracoBuilderExtensions).Assembly,
-                            AppCaches.NoCache, // Disable caches in integration tests
-                            testHelper.GetLoggingConfiguration(),
-                            // TODO: Yep that's extremely ugly
-                            (configs, umbVersion, ioHelper, logger, profiler, hostingEnv, backOfficeInfo, typeFinder, appCaches, dbProviderFactoryCreator) =>
-                            {
-                                var runtime = UmbracoIntegrationTest.CreateTestRuntime(
-                                    configs,
-                                    umbVersion,
-                                    ioHelper,
-                                    logger,
-                                    profiler,
-                                    hostingEnv,
-                                    backOfficeInfo,
-                                    typeFinder,
-                                    appCaches,
-                                    dbProviderFactoryCreator,
-                                    testHelper.MainDom,         // SimpleMainDom
-                                    dbInstallEventHandler);     // DB Installation event handler
+                () =>
+                {
+                    builder.Services.AddUmbracoCore(
+                        builder.WebHostEnvironment,
+                        container,
+                        typeof(UmbracoBuilderExtensions).Assembly,
+                        AppCaches.NoCache, // Disable caches in integration tests
+                        testHelper.GetLoggingConfiguration(),
+                        // TODO: Yep that's extremely ugly
```

Comment:
[minor] That "ugly" comment is also in the method itself. Probably can be removed